### ISSUE TITLE
Adding unique peak IDs

### ISSUE THREAD

**Issue by mlnresa** [LHentges/LanceOtron#12]
Hi, is there a way to add an extra column of unique peak IDs when calling peaks? I'm adding them manually now after running LanceOtron, but it would be nice to get it done in the same step.

**Comment by LHentges** [comment]
Just wanted to give you a quick update - this feature has been added, along with a peak summit column, to the latest version in PyPI (i.e. ```pip install lanceotron```). It should be version 1.2.6. 

We actually updated it a couple weeks ago now, but I've been holding back on a reply until conda was also ready. Unfortunately there  is a dependency conflict unique to conda, so this update has come slower than we would have liked, but fingers-crossed this should also be resolved soon!

A bit long-winded, but if you're using pip everything should be set, and if using conda hold tight for a little longer.
